OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyUtils.cs

```diff
@@ -257,23 +257,28 @@ public static void SetupEditableNameWidget(Widget parent, Session.Slot s, Sessio
 			name.IsDisabled = () => orderManager.LocalClient.IsReady;
 
 			name.Text = c.Name;
-			name.OnLoseFocus = () =>
+			name.OnEnterKey = () =>
```

Comment:
Removing the `OnLoseFocus` leaves the widget in an inconsistent state if you unfocus by clicking on the chat text field.